src/repl.jl

```diff
@@ -288,6 +288,15 @@ function insert_keybindings(repl = Base.active_repl)
     end
 
     main_mode.keymap_dict = LineEdit.keymap(Dict{Any, Any}[NEW_KEYBINDINGS, main_mode.keymap_dict])
+
+    # Inject ^R keybinding into other repl modes for fzf reverse search to work
+    for repl_mode in mirepl.interface.modes[2:end]
```

Comment:
I'm a bit worried this is too intrusive, ideally we only want this to work on Pkg and Shell mode right? not, say, R-mode or the now-gone C++ mode.